action/rename.php

```diff
@@ -52,6 +58,26 @@ public function handle_pagetools(Doku_Event $event) {
             array_slice($event->data['items'], $offset, null, true);
     }
 
+    /**
+     * Add 'rename' button to page tools, new SVG based mechanism
+     *
+     * @param Doku_Event $event
+     */
+    public function addsvgbutton(Doku_Event $event) {
+        global $INFO, $JSINFO;
+        if(
+            $event->data['view'] !== 'page' ||
+            !$this->getConf('pagetools_integration') ||
+            !$JSINFO['move_renameokay']
+        ) {
+            return;
+        }
+        if(!$INFO['exists']) {
+            return;
+        }
+        array_splice($event->data['items'], -1, 0, [new \dokuwiki\plugin\move\MenuItem()]);
```

Comment:
DokuWiki stable still requires only PHP 5.3 according to composer.json. Therefore, I would prefer to not to have the new array syntax in files that are also included in old DokuWiki versions. Apart from that 👍